inspirehep/utils/record_getter.py

```diff
@@ -109,5 +108,6 @@ def get_es_record_by_uuid(uuid):
 
 @raise_record_getter_error_and_log
 def get_db_record(pid_type, recid):
+    from inspirehep.modules.records.api import InspireRecord
```

Comment:
That import is a workaround due to we receive an import error if I import it at the top of the file.

`Original error message: 'cannot import name InspireRecord'.
Make sure your test modules/packages have valid Python names.`